### PR TITLE
Fix insecure mode

### LINKED ISSUE #3146: Insecure mode bug
**Describe the bug**
Setting environment `MOBILE_APP=true` or passing the url param `?insecure=true` should set session.insecureMode to true, but it doesn't.

**To Reproduce**

1. Execute `yarn serve:mobile`
2. Open Lunie in the browser
3. Go to Sign In
4. Go to Create a new address
5. Insecure warn is showed

**Expected behavior**
You should be able to create a new address without the warn.

### PR DESCRIPTION
**Pull request by mariopino**
Closes #3146 

**Description:**

Fix insecure mode handling.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/initializeApp.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ function setOptions(urlParams, store) {
     store.dispatch(`setNetwork`, { id: urlParams.network })
   }
 }
-
+`
```
Comment (faboweb) [inline src/initializeApp.js]: I think you added characters here

Comment (mariopino) [reply src/initializeApp.js]: > I think you added characters here

Yes, it was fixed 2 seconds later :-)

File: package.json [hunk 2]
```diff
@@ -49,6 +49,7 @@
     "cross-spawn": "7.0.1",
     "crypto-js": "^3.1.9-1",
     "graphql": "14.4.2",
+    "https-proxy-agent": ">=3.0.0",
```
Comment (jbibla) [inline package.json]: what does this have to do with insecure mode? maybe remove this?

**Comment by jbibla** [comment]
it seems like this is a good fix for insecure mode, but to be clear i don't think we want to use insecure mode on mobile. we should keep insecure mode (or is it called experimental now?) for insecure / experimental things and we should use the mobile env variable for mobile specific things. this is part of what i tried to change in #3125

**Comment by mariopino** [comment]
> it seems like this is a good fix for insecure mode, but to be clear i don't think we want to use insecure mode on mobile. we should keep insecure mode (or is it called experimental now?) for insecure / experimental things and we should use the mobile env variable for mobile specific things. this is part of what i tried to change in #3125

Thanks for re-clarification Jordan :-), I thought the opposite idea because that was the behaviour in feature/api-gateway.

File: package.json (hunk 2 above)
Comment (mariopino) [reply package.json]: Not sure why this has been end here :-/, but this module it's not currently used in develop so it's safe to remove it.

**Pull request merged**